pm/sender.go

```diff
@@ -74,6 +78,24 @@ func (s *sender) EV(sessionID string) (*big.Rat, error) {
 	return ticketEV(session.ticketParams.FaceValue, session.ticketParams.WinProb), nil
 }
 
+func (s *sender) Validate() error {
+	info, err := s.senderManager.GetSenderInfo(s.signer.Account().Address)
+	if err != nil {
+		return fmt.Errorf("could not get sender info: %v", err)
+	}
+
+	if info.Deposit.Cmp(big.NewInt((0))) == 0 {
```

Comment:
I don't think we need this deposit check since `validateTicketParams()` already has a deposit check (based on the `faceValue` required by O).

Reply:
The reason I included it was because we used to also just close the RTMP connection when B had no deposit.

fixed in 35979be